pandas/io/common.py

```diff
@@ -394,13 +395,13 @@ def _get_handle(path_or_buf, mode, encoding=None, compression=None,
     elif is_path:
         if compat.PY2:
             # Python 2
-            f = open(path_or_buf, mode)
+            f = io.open(path_or_buf, mode, newline='\n')
```

Comment:
Why do we need to use `io.open` for this one?

Reply:
I referenced [this stackoverflow article](https://stackoverflow.com/questions/34148441/python-2-7-newline-in-function), but it was not appropreate as all CI tests failed in py2 environment.
I will fix it.